src/library/scala/util/control/ControlThrowable.scala

```diff
@@ -10,30 +10,38 @@
  * additional information regarding copyright ownership.
  */
 
-package scala
-package util.control
+package scala.util.control
 
-/** A marker trait indicating that the `Throwable` it is mixed into is
- *  intended for flow control.
+/** A parent class for exceptions intended for flow control.
  *
- *  Note that `Throwable` subclasses which extend this trait may extend any
- *  other `Throwable` subclass (eg. `RuntimeException`) and are not required
- *  to extend `Throwable` directly.
- *
- *  Instances of `Throwable` subclasses marked in this way should not normally
- *  be caught. Where catch-all behaviour is required `ControlThrowable`
- *  should be propagated, for example:
+ *  Instances of `ControlThrowable` should not normally
+ *  be caught. Where catch-all behaviour is required, `ControlThrowable`
+ *  should be propagated; for example:
  *  {{{
  *  import scala.util.control.ControlThrowable
  *
  *  try {
  *    // Body might throw arbitrarily
  *  } catch {
  *    case c: ControlThrowable => throw c // propagate
- *    case t: Exception        => log(t)  // log and suppress
+ *    case t: Exception        => log(t)  // log and ignore
```

Comment:
This is misleading if CT is not Exception.

Reply:
yeah, it should probably be changed or removed I guess?